### PR TITLE
Fix timestampable extension

### PR DESCRIPTION
**Pull request by TomasPilaru**
Hi, 

in latest KnpLabs/DoctrineBehaviors package is new required option called 'db_field_type'.

See [KNP documentation](https://github.com/KnpLabs/DoctrineBehaviors#timestampable).

Thanks!

### CODE REVIEW

File: tests/DI/DoctrineBehaviorsExtensionTest.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ class DoctrineBehaviorsExtensionTest extends PHPUnit_Framework_TestCase
 	/**
 	 * @var int
 	 */
-	const LISTENER_COUNT = 18;
+	const LISTENER_COUNT = 19;
```
Comment (enumag) [inline tests/DI/DoctrineBehaviorsExtensionTest.php]: What caused this change? Is it dependenf on the version of the KnpLabs package?

File: src/DI/TimestampableExtension.php [hunk 2]
```diff
@@ -37,7 +38,8 @@ public function loadConfiguration()
 			->setClass(TimestampableSubscriber::class, [
 				'@' . $this->getClassAnalyzer()->getClass(),
 				$config['isRecursive'],
-				$config['trait']
+				$config['trait'],
+				$config['dbFieldType']
```
Comment (enumag) [inline src/DI/TimestampableExtension.php]: Add a comma to the end of this line.

File: tests/DI/DoctrineBehaviorsExtensionTest.php (hunk 1 above)
Comment (enumag) [inline tests/DI/DoctrineBehaviorsExtensionTest.php]: Actually this was changed in the master branch already, see https://github.com/Zenify/DoctrineBehaviors/commit/ead0c88ea20a2dc9d1d3153207873c1a2a036fc7.

Please rebase this PR to the current master.

**Comment by TomasPilaru** [comment]
Rebased and fixed.

**Comment by enumag** [comment]
Sqash it to a single commit please.

**Comment by TomasPilaru** [comment]
Done

**Comment by enumag** [comment]
Rebase to master and remove the second commit. It is not necessary because of https://github.com/Zenify/DoctrineBehaviors/commit/3fa694e57d5a808d72d614535bff2ae7d8d58afa.

**Comment by TomasPilaru** [comment]
I edited the last commit with compose.json to support 1.2 version

Between latest version of KnpLabs/DoctrineBehaviors (which is 1.3.0) and 1.2.0 is substantial difference in ctor. See [1.3.0 ctor](https://github.com/KnpLabs/DoctrineBehaviors/blob/1.3.0/src/ORM/Timestampable/TimestampableSubscriber.php#L32) and [1.2.0 ctor](https://github.com/KnpLabs/DoctrineBehaviors/blob/1.2.0/src/ORM/Timestampable/TimestampableSubscriber.php#L31). There is a new mandatory parameter called `dbFieldType`. I add this parameter in commit ab81a71f4fbcb015314463b6585411d704fd002e.

Rebase should be done, or am I wrong?

**Comment by enumag** [comment]
I know about the ctor change but unless nette/di is stupid your updated DI extensions should work for both.

Yeah I already told you to rebase.

**Comment by TomasPilaru** [comment]
It's already rebased and tested.

**Comment by enumag** [comment]
@TomasPilaru Thank you!

**Comment by TomasPilaru** [comment]
@enumag You're welcome! Could you release new tag?

**Comment by enumag** [comment]
@TomasPilaru Done. I'll have to be more careful wil future pull requests though. I shouldn't have merged #11. And your second commit has a wrong commit message (it allows v1.3, not v1.2).

**Comment by TomasPilaru** [comment]
@enumag Thanks! Yeah, sorry for a commit message...

**Comment by TomasPilaru** [comment]
@enumag Packagist is outdated (https://packagist.org/packages/zenify/doctrine-behaviors) is there some  delay or github hook failed?

**Comment by enumag** [comment]
@TomasVotruba Can you give me access to packagist for Zenify and Symnedi?

**Comment by enumag** [comment]
Actually I should probably ask @f3l1x?

**Comment by f3l1x** [comment]
@enumag Fixed. :)

**Comment by TomasPilaru** [comment]
@enumag @f3l1x Cool! So fast, thanks :)

**Comment by enumag** [comment]
Ok, packagist stopped working because @TomasVotruba whose API token for packagist was used on GitHub is not a maintainer anymore. Which means I have to fix the hook for all Symnedi and Zenify packages. I've updated the package manually this time but the hook should work again for future releases.

@f3l1x Can I ask for access to other Zenify packages as well? :-)

**Pull request merged**